lib/Context/Process.php

```diff
@@ -120,7 +120,7 @@ public function __construct($script, string $cwd = null, array $env = [], string
 
         $command = \implode(" ", [
             \escapeshellarg($binary),
-            $phpConfig ? "-n -c $phpConfig" : "",
+            $phpConfig ? \escapeshellarg("-n -c $phpConfig") : "",
```

Comment:
That'll make `-n -c $phpConfig` one argument. The `escapeshellarg` needs to be only around the `$phpConfig`.

Reply:
Must be
```php
"-n -c ".\escapeshellarg("$phpConfig")
```